### PR TITLE
FD + aqua + Booster updates

### PR DESCRIPTION
**Pull request by iderdik**
Greetings and thanks for the PR.  We have a few rules so that everyone enjoys your Pull Request.

## Changelog

- Updates default Booster port for FD
- Sends Aqua vars to FD
- New FD (auto-create database, Aqua integration, bug fixes, `/v2/health/check` endpoint)

## Notes

You must explain how any `docker run` executions your PR introduces handles log accumulation. 

Did you introduce any command that executes `docker run`? - No

Are there any dependencies on github.com/adobe-platform/infrastructure? -No

If so:
 
Is an update to the base stack required(rare)? - No

Is an A/B required? - Yes

Are you dependent on new secrets, or infrastructure in any way? - Yes, needs Aqua endpoint,  username and password to be passed in

### CODE REVIEW

File: v1/util/flight-director.sh [hunk 1]
```diff
@@ -42,4 +53,8 @@ fi
   -e FD_ALLOW_MARATHON_UNVERIFIED_TLS=`etcdctl get /flight-director/config/allow-marathon-unverified-tls` \
   -e FD_SCALER_PROTOCOL=`etcdctl get /flight-director/config/scaler-protocol` \
   -e FD_SCALER_ENDPOINT=$SCALER_ENDPOINT \
+  -e FD_AQUA_PROTOCOL=AQUA_PROTOCOL \
+  -e FD_AQUA_ENDPOINT=AQUA_ENDPOINT \
+  -e FD_AQUA_USER=AQUA_USER \
+  -e FD_AQUA_PASSWORD=AQUA_PASSWORD \
```
Comment (matthewdfuller) [inline v1/util/flight-director.sh]: These need `$`

**Comment by matthewdfuller** [comment]
👍

**Comment by phylake** [comment]
@adobe-platform/ops This is needed for min/max containers to be passed through to booster from FD

**Comment by iderdik** [comment]
I have one more small change to make before merging

**Comment by iderdik** [comment]
OK @matthewdfuller - merge when ready

**Pull request merged**